odh-dashboard/base/routes.yaml

```
@@ -4,14 +4,14 @@ metadata:
   annotations:
     kubernetes.io/tls-acme: 'true'
     haproxy.router.openshift.io/hsts_header: max-age=31536000;includeSubDomains;preload       
-  name: rhods-dashboard
+  name: odh-dashboard
 spec:
   port:
-    targetPort: 8080
+    targetPort: 8443
   tls:
     insecureEdgeTerminationPolicy: Redirect
-    termination: edge
+    termination: Reencrypt
```

```suggestion
    termination: reencrypt
```

The accepted values are `edge`, `passthrough` and `reencrypt`. All other values are silently ignored.

---

mmm let me change that in upstream too then!

---

done